lib/workers/global/initialize.ts

```diff
@@ -93,4 +94,5 @@ export async function globalInitialize(
 
 export async function globalFinalize(config: RenovateConfig): Promise<void> {
   await packageCache.cleanup(config);
+  resetGlobalLogLevelRemaps();
```

Comment:
What does this achieve? Aren't we exiting anyway?

Reply:
Just to clarify intent 🤷‍♂️